tests/core/conftest.py

```diff
@@ -38,6 +38,12 @@
 
 END_TO_END_STORY_FILE = "data/test_evaluations/end_to_end_story.md"
 
+RETRIEVAL_INTENT_STORY_FILE = "data/test_evaluations/retrieval_intent_story.yml"
```

Comment:
Seems like we're using the `RETRIEVAL_INTENT_STORY_FILE` and `RETRIEVAL_INTENT_WRONG_PREDICTION` only once, probably that doesn't make sense to extract them as constants, and just reference them in the actual tests that are using them.